src/cardanoLauncher.ts

```diff
@@ -486,8 +486,12 @@ async function walletExe(
         '' + config.nodeConfig.restPort,
       ]);
     case 'byron':
+      const networkArg = (config.networkName === 'testnet')
```

Comment:
Given `mainnet` is the exception, shouldn't the logic be the following?:

```
const networkArg = (config.networkName !== 'mainnet')
        ? ['--testnet', config.nodeConfig.network.genesisFile]
        : [`--${config.networkName}`]
```

This way `self-node` is treated as _a testnet_